pipelines/smartseq2_single_sample/README.md

```diff
@@ -126,7 +128,7 @@ The [Picard task](https://github.com/HumanCellAtlas/skylab/blob/master/library/t
 
 ### Part 2: Transcriptome Quantification Tasks
 #### 2.1 Align reads to the transcriptome using HISAT2
-The [HISAT2RSEM task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) uses HISAT2 to align reads to the reference transcriptome. The task requires the hisat2_ref_trans_index file and the sample fastq files as input. The output of this task is a transcriptome-aligned BAM file and an alignment log file.
+The [HISAT2RSEM task](https://github.com/HumanCellAtlas/skylab/blob/master/library/tasks/HISAT2.wdl) uses HISAT2 to align reads to the reference transcriptome. The task requires the hisat2_ref_trans_index file and the sample FASTQ files as input. The output of this task is a transcriptome-aligned BAM file and an alignment log file.
```

Comment:
Another non-relative path. Looking at this document, there are many non-relative paths used. Is one form preferred over the other? To me, relative paths look like they'd be easier to edit and maintain. If we do want to move to non-relative paths, maybe a separate JIRA task is needed to fix up all the cases in our current docs. Alternatively, if full paths are preferred, then we shouldn't be moving some references to relative paths.

Reply:
I have been trying to move to relative paths as I'm cleaning up docs. I just did another pass through this one and hopefully got them all!